### ISSUE TITLE
Object Designer does not show Standard objects

### ISSUE THREAD

**Issue by AOBoga** [martonsagi/al-object-designer#102]
Hello,

I have installed AL Object Desinger in VS Code.  I have successfully executed "Download Symbols" command. However, AL Object Desinger does not sho all Standard Objects. I have been using the al object desginer for 2 years and this is the first time, that I face this problem. 

![Example Screenshot](https://github.com/martonsagi/al-object-designer/assets/145163773/c037851f-9a46-4b1e-9e40-5e94e1df635c)

It begins from object 870... What could be the reasons???

Thank you and best regards

**Comment by epic-Abby21** [comment]
I'm having the same issue.

**Comment by chogio** [comment]
same here, first time having this issue after using for so many years

**Comment by tgotzen** [comment]
same here

**Comment by OseChris** [comment]
It stopped working with new symbols bc > v23. I bet its the namespaces additions AL object designer cant parse nor handle.

**Comment by elweilando** [comment]
Yes, it is the namespaces. The objects that are still displayed do not have any.

**Comment by marjuro28** [comment]
same here, I can't see the standard objects

**Comment by bchayer** [comment]
Same here

**Comment by DaggiLiehrSowis** [comment]
I have the same problem. It would be fantastic if the problem could be fixed soon. Without the Object Designer, programming is only half as fun ;-)

**Comment by DennisHulverscheidt** [comment]
Same here
